client/src/lib/transformers/dataset.ts

```diff
@@ -1,4 +1,4 @@
-import _ from "lodash";
+import { omit } from "lodash";
```

Comment:
Réfléxion générale

Lodash est un paquet très lourd (voir bundlephobia : https://bundlephobia.com/package/lodash@4.17.21) et je remarque qu'il est uniquement utiliser pour utiliser la fonction `omit`

De manière générale, je préconise de ne pas utiliser lodash sur les projets JS. Pour moi ES6  a apporté des fonctionalités  built-in qui ne rendent plus si nécessaire que ça l'utilisiation de lodash

Ma question est : est-ce que l'on a vraiment besoin de ce paquet ?

Si oui, il faudrait favoriser le treeshaking en important uniquement les utilitaire lodash dont on a besoin (comme ici)